setup.sh

```diff
@@ -89,10 +89,8 @@ install_homebrew () {
 install_rbenv_linux() {
     msg "Installing Ruby..."
     sudo apt update
-    git clone git://github.com/sstephenson/rbenv.git ~/.rbenv
-    git clone git://github.com/sstephenson/ruby-build.git ~/.rbenv/plugins/ruby-build
-
-    sudo apt-get install -y libreadline-dev
+    git clone git://github.com/rbenv/rbenv ~/.rbenv/
+    git clone git://github.com/rbenv/ruby-build ~/.rbenv/plugins/ruby-build
```

Comment:
Should we change these to use http? That is less likely to cause issues, as everyone should have http access if they have an internet connection.